WISE-Community/WISE
# Notebook grading redesign

Redesigned the notebook grading view in the Classroom Monitor to show a sortable list of workgroups (similar to node grading view). Notebook work (notes and/or reports, if enabled) can be toggled opened and closed for each workgroup.

To test:
- Open teacher tools for a run with student notebook enabled.
- Navigate to Notebook view and ensure that viewing student notebooks (both the notes and report tabs) works properly.
- Enable and disable notes and reports in the run. Make sure that list column headers and the work displays for each are shown when enabled and not shown when disabled.

A few related issues:
1. I've hidden the ability to score and send comments for notebook reports since this functionality is currently broken. Creating a separate issue to address.
2. Notebook work doesn't update if students change their work while the teacher tools are open. You have to reload the teacher tools to see any new notebook content. Also creating a separate issue for this.

Closes #2318.
